pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -67,27 +65,15 @@ func IsReadyToReconcile(c client.Client, clustercontext *clusterd.Context, names
 
 	logger.Debugf("%q: CephCluster resource %q found in namespace %q", controllerName, cephCluster.Name, namespacedName.Namespace)
 
-	// If the cluster is healthy
-	// Test a Ceph command to verify the Operator is ready
-	// This is done to silence errors when the operator just started and cannot reconcile yet
-	status, err := cephclient.Status(clustercontext, namespacedName.Namespace)
-	if err != nil {
-		if strings.Contains(err.Error(), "error calling conf_read_file") {
-			logger.Infof("%q: operator is not ready to run ceph command, cannot reconcile yet.", controllerName)
-			return cephCluster, false, cephClusterExists, WaitForRequeueIfCephClusterNotReady
+	// read the CR status of the cluster
+	if cephCluster.Status.CephStatus != nil {
```

Comment:
Is this change needed for the object store user? Or it's just removing the unnecessary call to retrieve the status since it's already available?

Reply:
It's needed since when we do an external cluster we cannot quite call the `Status` function. If we want to do this, we would need to instruct this function which user to use so that we can properly connect to the cluster. See we have `Status` and `StatusWithUser()`

I thought that relying on the CR status was good enough.